db/models/users.model.js

```diff
@@ -0,0 +1,50 @@
+const { DataTypes } = require("sequelize");
+module.exports = (sequelize) => {
+  const User = sequelize.define("users", {
+    id: {
+      type: DataTypes.UUID,
+      unique: true,
+      defaultValue: DataTypes.UUIDV4,
+    },
+    email: {
+      type: DataTypes.STRING,
+      unique: true,
+      allowNull: false,
+      primaryKey: true,
```

Comment:
Set this to false and let the ID only be the primary key. Happy to see you already made email unique.